### ISSUE TITLE
Auto hide title shifting

### ISSUE THREAD

**Issue by nguyenkien** [johnfactotum/foliate#298]
This is how it look right now. 
![eee](https://user-images.githubusercontent.com/1940081/79063845-808c3180-7cce-11ea-9b29-6e5d7d973be1.gif)

I suggest add add this css to fix it

`.autohide-label {padding-bottom: 1px}`

![fix](https://user-images.githubusercontent.com/1940081/79063850-8e41b700-7cce-11ea-9ed8-24d7160db6ad.gif)

**Comment by johnfactotum** [comment]
This depends on the GTK theme and font. In some setups they actually do line up now, and adding that padding would cause shifting.

The only way to get the same style as the headerbar is to use an actual headerbar. But even then it's not really guaranteed. Overriding the style to remove any borders and match the book's colours would also be a lot harder if we use a headerbar instead of a label.

Furthermore, the headerbar has buttons, while the label must span the full width of the window (otherwise, if we use transparent dummy buttons, it would look very weird, with the text ellipsed and not centered despite having enough space left), so they will never line up perfectly when the title is long.

Another alternative is to not show any title on the overlaid headerbar at all. The downside is, of course, no title on the headerbar.

In short, it's not really possible to line them up perfectly, and I think the current behavior is good enough.
